### PR TITLE
2 database design

### PR DESCRIPTION
**Pull request by Mast3rwaf1z**
there's a little implementation of addition to the database, but this PR is primarily database test design and initialization of the database.

### CODE REVIEW

File: include/Table.hpp [hunk 1]
```diff
@@ -43,14 +43,48 @@ class Table {
...
+                query.exec();
+            }
+            catch(SQLite::Exception e){
+                std::string msg;
+                switch(e.getErrorCode()){
+                    case 19:
+                        msg = "Error! Unique keyword not followed!";
+                        break;
+                    default:
+                        msg = "Unknown error!";
```
Comment (Taoshix) [inline include/Table.hpp:56]: maybe print e.getErrorCode() here too

File: include/Table.hpp [hunk 2]
```diff
@@ -43,14 +43,48 @@ class Table {
...
+                }
+            }
+            catch(SQLite::Exception e){
+                std::string msg;
+                switch(e.getErrorCode()){
+                    case -1:
+                        msg = std::format("Error! no value of '{}' in column: '{}' of table: '{}' exists", value, key, this->name);
+                        break;
+                    default:
+                        msg = "Unknown error!";
```
Comment (Taoshix) [inline include/Table.hpp:80]: see first comment

**Review by Taoshix — APPROVED** [review]
only minor things, but still good enough

File: include/Table.hpp (hunk 1 above)
Comment (Mast3rwaf1z) [reply include/Table.hpp:56]: added it to the print string, no reason to add it here

**Pull request merged**